lib/Migration/Version20000Date20240716190335.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version20000Date20240716190335 extends SimpleMigrationStep {
+	/**
+	 * @param IOutput $output
+	 * @param Closure(): ISchemaWrapper $schemaClosure
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		$table = $schema->getTable('talk_invitations');
+		if (!$table->hasIndex('talk_user_state_idx')) {
+			$table->addIndex(['user_id', 'state'], 'talk_user_state_idx');
```

Comment:
Not a problem, but just so you know. PostgreSQL is automatically adding an IDX suffix. Oracle has another one. So we just skip them usually :D
```suggestion
		if (!$table->hasIndex('talk_user_state')) {
			$table->addIndex(['user_id', 'state'], 'talk_user_state');
```

Reply:
Did not know that, thanks for letting me know! Just updated the name